geospaas/base_viewer/forms.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib.gis import forms
+from django.utils import timezone
+from datetime import tzinfo
+from leaflet.forms.widgets import LeafletWidget
+import pytz
+from geospaas.catalog.models import Source
+
+
+class BaseSearchForm(forms.Form):
+
+    polygon = forms.GeometryField(label='choose a location (or leave it empty for all locations)',
+                                  widget=LeafletWidget(attrs={
+                                      'settings_overrides': {
+                                          'DEFAULT_CENTER': (60.0, 5.0),
+                                          'DEFAULT_ZOOM': 1,
+                                          'PLUGINS': {'forms': {'auto-include': True}},
+                                      }
+                                  }))
+    utc = pytz.utc
```

Comment:
This should not be a class attribute, can you directly use `timezone.utc` as the `tzinfo` argument in `time_coverage_start`?

Reply:
Yes, done.